pkg/apis/workflow/v1alpha1/workflow_types.go

```diff
@@ -710,6 +721,11 @@ type VolumeGC struct {
 	Strategy VolumeGCStrategy `json:"strategy,omitempty" protobuf:"bytes,1,opt,name=strategy,casttype=VolumeGCStrategy"`
 }
 
+// GetStrategy returns the VolumeGCStrategy to use for the workflow
+func (vgc VolumeGC) GetStrategy() VolumeGCStrategy {
+	return vgc.Strategy
```

Comment:
and if empty? return default?

Reply:
Will do.  My thought was that defining an empty VolumeGC block was likely an error, however returning `OnWorkflowSuccess` as a sane default here makes sense.